python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -176,6 +179,10 @@ class HistPanel(Config):
         "default stats: N, median, sigma mad are shown",
         default=None,
     )
+    addThreshold = Field[bool](
+        doc="Read in the predefined threshold and add it to the histogram.",
```

Comment:
Should this be "threshold(s)"? Aren't there often more than one thresholds per metric?
Also, suggest "indicate it on the histogram" rather than "add".